Update bulma to 0.8.2

illogic-al opened the pull request:
Updates bulma to new 0.8.2 version

### Tradeoffs
Form has been taken out of elements and added to its own scss file

### Testing Done
Not yet. Creating forked npm and meteor packages in order to test in app.

illogic-al committed:
Update bulma to 0.8.2.

- Anyone moving from 0.7 to 0.8 will have to take note of the moving of the form styles.
- Also update npm packages to more secure versions.

illogic-al committed:
Update docs sass to 0.8.2.

bduff9 commented on README.md:
```diff
@@ -6,12 +6,14 @@ This is the unofficial scss version of Bulma.  For all questions or concerns abo
 
-[![npm](https://img.shields.io/npm/v/bulma_scss.svg)](https://www.npmjs.com/package/bulma_scss)
-[![npm](https://img.shields.io/npm/dm/bulma_scss.svg)](https://www.npmjs.com/package/bulma_scss)
+[![npm](https://img.shields.io/npm/v/bulma.svg)][npm-link]
```
@illogic-al Can you please update this to this project's badges?

illogic-al replied:
How would I do that? I don't know what those are?

bduff9 replied:
Well, I'm not sure if you are familiar with pull requests, however all diffs can be seen here.  This is what I am reviewing.  So you can see the lines above that you replaced point to bulma_scss badges, and you replaced them with bulma badges.  Please undo this change

illogic-al replied:
Ah, I see. Replaced your project's images (badges) with the bulimia project's. Yup, I can fix that.

illogic-al replied:
OK. Changes have been pushed.

bduff9 commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bulma_scss",
-  "version": "0.7.2-rc.0",
+  "version": "0.8.2",
```
@illogic-al Please use an rc version here, if this needs to be updated at all, we would have to go to 0.8.3 which would prevent this from lining up with bulma.

illogic-al replied:
There are no rc versions. Do you want me to just modify the version number manually?

bduff9 replied:
I don't think you understand.  Just curious, have you ever released a package to npm before?  What I am referencing is you updated the version of this package to the same version as the bulma package.  While this is fine, if there are any mistakes or issues in this release, we would then need to update this package to 0.8.3 to re-release.  This would cause this package to not line up to the current bulma and even worse, when bulma comes out with 0.8.3, we cannot use that to update.

Again, the diff here shows what I mean, we were on 0.7.2-rc.0 (which means 0.7.2 of Bulma) but it is a rc (release candidate).  Once released, if we decided that everything was perfect, we could then release the rc as an actual version, but haven't really felt the need to do so as rc's can be used just like other releases.

illogic-al replied:
Nope, haven't ever released my own package to npm till yesterday to test my changes.
Thanks for the explanation. Confusion resolved. 
I'll make these two changes requested post-haste!

illogic-al replied:
OK. Changes have been pushed.

illogic-al committed:
Updates made per pull request review.

- Modify npm package version per PR.
- Undo change that replaced bulma_scss project badges with official bulma badges.

bduff9 reviewed (approved):
Looks better, thanks!

illogic-al commented:
FYI, I had issues when creating the meteor package. The test scss files used in a node dependency were causing issues. 
I went into the node_modules folder, deleted those test files, and then the fournova scss module was able to finish doing its thing. 
I don’t know if there is a “proper” way to fix that, but being new, I just assumed the files fournova’s    module was choking on should not have been part of the npm distributed bundle.  🤷🏽‍♂️

bduff9 commented:
@illogic-al Yeah they should probably not be in there.  Feel free to open another PR if you'd like to fix this.

illogic-al commented:
This wasn't an issue in the bulma_css package it was in some downstream dependency of the fournova package. It's a bit beyond my skill level to figure out what the actual issue is. 
I was just passing on the info since I assumed you'd want to update your meteor package. Might help you retain some hair :-)
